forge-gui/res/cardsfolder/upcoming/relive_the_past.txt

```diff
@@ -0,0 +1,11 @@
+Name:Relive the Past
+ManaCost:5 G W
+Types:Sorcery
+A:SP$ Pump | TgtZone$ Graveyard | ValidTgts$ Artifact | TargetMin$ 0 | TargetMax$ 1 | SubAbility$ DBLand |  TgtPrompt$ Select up to one target artifact from your graveyard | SpellDescription$ Return up to one target artifact card, up to one target land card, and up to one target non-Aura enchantment card from your graveyard to the battlefield. 
+SVar:DBLand:DB$ Pump | TgtZone$ Graveyard | ValidTgts$ Land | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Select up to one target land from your graveyard | SubAbility$ DBEnchantment
+SVar:DBEnchantment:DB$ Pump | TgtZone$ Graveyard | SubAbility$ DBChangeZoneAll | ValidTgts$ Enchantment.nonAura | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Select up to one target non-aura enchantment from your graveyard 
+SVar:DBChangeZoneAll:DB$ ChangeZone | Origin$ Graveyard | GainControl$ True | Destination$ Battlefield | Defined$ Targeted | AnimateSubAbility$ DBAnimate
```

Comment:
why GainControl?

Reply:
I think because it allows you to target permanents in an opponents graveyard ?